test/unit/streaming.MediaPlayerSpec.js

```diff
@@ -260,297 +259,6 @@ describe('MediaPlayer', function () {
                 expect(player.setAutoPlay.bind(player, 12)).to.throw(Constants.BAD_ARGUMENT_ERROR);
             });
 
-            it('should not set setMaxAllowedBitrateFor value if it\'s not a number type or NaN or if type is not Video or Audio', function () {
```

Comment:
a lot of this unit tests have to be saved to be sure that settings type is correct.

Reply:
Yep. That's part of the pending work I mentioned and that will be included in a third PR.